docs/editing-components.md

```diff
@@ -0,0 +1,376 @@
+     [used in `InfoUnitGroup`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/atomic_elements/organisms.py#L54),
+     among other places.
+1. Add any desired parameters:
+   - `required=False` if you do _not_ want the field to be required
+     (it usually is, by default)
+   - `label='Some label'` if you would like the editor to show a label more
+     meaningful than the sentence-case transformation of the field name
+   - `help_text='Some text'` if the field needs a more verbose explanation to
+     be shown in the editor to make it clear to users how it should work
+   - `default=<some appropriate value>` if you want the field to have a
+     specific default value, e.g., `True` to have a `BooleanBlock` checkbox
+     default to checked.
+   - Certain blocks may take other arguments, as described in the
+     [basic Wagtail blocks documentation](https://docs.wagtail.io/en/v1.13.4/topics/streamfield.html#basic-block-types).
+1. [Edit the component template](#the-html-template) to do something with the
+   field's data – output it, use it to trigger a CSS class, etc.
+1. [Create a schema migration.](#schema-migrations)
+
+#### Removing a field
+
+These instructions presume that you do not care
+about any data stored in the field you are deleting.
+If that is not the case, please skip to the
+[instructions for editing a field](#editing-a-field)
+and come back here when instructed.
+
+1. Locate the field you want to remove in the block's Python class, either in
+   [atoms.py](https://github.com/cfpb/cfgov-refresh/tree/master/cfgov/v1/atomic_elements/atoms.py),
+   [molecules.py](https://github.com/cfpb/cfgov-refresh/tree/master/cfgov/v1/atomic_elements/molecules.py),
+   or [organisms.py](https://github.com/cfpb/cfgov-refresh/tree/master/cfgov/v1/atomic_elements/organisms.py).
+1. Delete the field definition.
+1. [Create a schema migration.](#schema-migrations)
+
+#### Editing a field
+
+1. [Determine if the change you want to make will need a data migration.](#you-may-also-need-a-data-migration)
+   - If the answer is **no**: make your changes,
+     [create a schema migration](#schema-migrations), and be on your merry way.
+   - If the answer is **yes**: continue on.
+1. [Add the new version of the field.](#adding-a-field)
+1. [Create a schema migration](#schema-migrations) for adding the new field.
+1. [Create a data migration](../wagtail-migrations/)
+   to copy data from the old field into the new field.
+1. [Edit the component template](#the-html-template)
+   to use the new field's data instead of the old field's data.
+1. [Remove the old field.](#removing-a-field)
+1. [Create a schema migration](#schema-migrations) for removing the old field.
+
+
+### Creating migrations for StreamField blocks
+
+**tl;dr:**
+
+1. `./cfgov/manage.py makemigrations -n <description_of_changes>`
+1. [Determine if a data migration is needed](#you-may-also-need-a-data-migration)
+
+#### Schema migrations
```

Comment:
It would be nice to link to the [Django documentation](https://docs.djangoproject.com/en/1.11/topics/migrations/) somewhere here.

Reply:
Sorta addressed in a47039c by moving most of it to the Wagtail Migrations page.